flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/common/config/JedisClusterConfigTest.java

```diff
@@ -46,4 +48,20 @@ public void shouldThrowIllegalArgumentExceptionIfNodeValuesAreEmpty(){
             .setNodes(set)
             .build();
     }
+
+    @Test
+    public void shouldSetPasswordSuccessfully() {
```

Comment:
could you add a test with no password setted.

Reply:
I added the test.